src/style.scss

```diff
@@ -1,4 +1,5 @@
 $toolbarHeight: 50px;
+$maxCaptionHeight: 150px;
```

Comment:
also, this was randomly selected - I think it is a good max size for a description.